packages/core/__tests__/diff/directive-usage.test.ts

```diff
@@ -690,7 +713,7 @@ describe('directive-usage', () => {
       `);
 
       const changes = await diff(a, b);
-      const change = findFirstChangeByPath(changes, 'Foo.external');
+      const change = findFirstChangeByPath(changes, '.@external');
```

Comment:
This indicates the directive is applied to the schema object. This `.` is necessary to distinguish directive usages from directive definitions at the root level.